ch09/ex-create-element.md

```diff
@@ -136,3 +136,66 @@ by using `insertBefore` with a reference node of null.
                TextLayout(x=13, y=35.25, width=48, height=12, word=More)
                TextLayout(x=73, y=35.25, width=84, height=12, word=content)
                InputLayout(x=169, y=35.25, width=200, height=12, node=<input>)
+
+
+
+Test: Screen redraws after
+
+Test `appendChild`
+
+    >>> script = """
+    ... new_elt = document.createElement("input");
+    ... my_p = document.querySelectorAll('p')[0];
+    ... my_p.appendChild(new_elt);
+    ... """
+    >>> js_url = wbemocks.socket.serve(script)
+    >>> body = "<script src=" + str(js_url) + "></script>" + body
+    >>> html_url = wbemocks.socket.serve(body)
+    >>> this_browser.new_tab(browser.URL(html_url))
```

Comment:
I don't think this works, because the `<script>` here would be run during page load, and that already causes a screen redraw. Here's what I propose:
- First, load `html_url` but don't add the script to it.
- Then do `this_browser.active_tab.js.run(script)`
- Then print `this_browser.active_tab.document`
This way the second step can only redraw the page if `appendChild` does.